terraform/azure/infra/main.tf

```diff
@@ -151,14 +151,43 @@ resource "azurerm_web_application_firewall_policy" "wandb" {
   }
 }
 
+resource "azurerm_network_security_group" "wandb" {
+  # The AKS Application Gateway ingress controller is only able to use a private IP
+  # if the gateway is using Standard_v2 or WAF_v2.  They both currently require
+  # a public IP, see: https://github.com/Azure/application-gateway-kubernetes-ingress/issues/741
```

Comment:
Is this still true after 2 years? This comment seems to suggest it would be resolved a few months after that issue was created in 2020: https://github.com/Azure/application-gateway-kubernetes-ingress/issues/741#issuecomment-589891004

Reply:
Yep, still the case.  Tried setting up a Standard_v2 load balancer with only a private IP and the Azure API rejected it.  I then used the Standard_v1 sku and the application gateway blew up when it attempted to update the gateway 🙃 .